vnext/Desktop/module.g.cpp

```diff
@@ -6,9 +6,9 @@
 #include "pch.h"
 #include "winrt/base.h"
 void* winrt_make_Microsoft_Internal_TestController();
-void* winrt_make_Microsoft_ReactNative_CompositionRootView();
-void *winrt_make_Microsoft_ReactNative_Composition_CompositionContextHelper();
-void *winrt_make_Microsoft_ReactNative_Composition_CompositionUIService();
+// void* winrt_make_Microsoft_ReactNative_CompositionRootView();//TODO: revert
```

Comment:
?

Reply:
Yeah, I meant to ask about this.

Integration tests (Desktop) won't compile unless this is commented out.
Clearly there's some import/setup missing.

Are those types meant to be part of the Desktop DLL?

Reply:
Yes, those types are part of the desktop dll